### ISSUE TITLE
Unable to Select Update Channel

### ISSUE THREAD

**Issue by AJ10015130** [MultiMC/Launcher#4395]
### Operating System

Windows

### Description of bug

The box should let me select what update channel I want, but instead it is greyed out and I can't click on it. I don't have any instances open, just Multimc.

![Screenshot (1)](https://user-images.githubusercontent.com/96843155/147699342-56c692de-784a-4904-ba8e-53de95ad584d.png)
.

### Steps to reproduce

Ran Multimc like usual.
Went into settings.
Tried to click on the box, but couldn't.

### This issue is unique

- [X] I have searched the issue tracker and did not find an issue describing my bug.

**Comment by phit** [comment]
please provide your launcher log, `MultiMC-0.log` in the main launcher folder

**Comment by AJ10015130** [comment]
[MultiMC-0.log](https://github.com/MultiMC/Launcher/files/7790487/MultiMC-0.log)
 
Is this the right thing?

**Comment by phit** [comment]
yup, looks like you are offline or something is blocking internet access to multimc completely, no network request is going through to multimc, microsoft or the analytics

**Comment by AJ10015130** [comment]
That's odd, I have an internet connection to my PC, so do you have any suggestions on where I can connect MultiMC to the internet?

**Comment by AJ10015130** [comment]
I've also noticed that a couple days ago the launcher started to ask me to choose an offline player name when I opened instances, preventing me from playing on multiplayer servers and it's been doing it ever since. So I'm guessing that's because of the internet issue as well.

**Comment by phit** [comment]
are you using any weird third party antivirus or similar?

**Comment by AJ10015130** [comment]
I think so? I believe it's the Norton antivirus software.

**Comment by phit** [comment]
it is probably blocking stuff

**Comment by AJ10015130** [comment]
Is there a place in MultiMC or in settings where I can get it unblocked? I'm sorry, I'm just new to this kind of stuff.

**Comment by phit** [comment]
no clue, I'm sure Norton has exclusion settings, this is out of scope for MultiMC

**Comment by AJ10015130** [comment]
Oh ok, I'll look into it. Thank you so much!
